docs/users/extend/additional-services.md

```diff
@@ -16,6 +16,6 @@ This recipe adds an Apache Solr 5.4 container to a project. It will setup a solr
 
 **Interacting with Apache Solr**
 
-- The Solr admin interface will be accessible at http://<projectname>.ddev.local:8983
-- The host connection for apache solr will be <projectname>.ddev.local
+- The Solr admin interface will be accessible at `http://<projectname>.ddev.local:8983/solr/`
+- To access Solr from app container use `http://sorl:8983/solr/`
```

Comment:
This is all good except the s/sorl/solr/ :)

I think you mean "To access the solr container from the web container"?